includes/definitions/etherwan.yaml

```diff
@@ -0,0 +1,10 @@
+os: etherwan
+text: 'Etherwan Managed Switch'
+type: network
+icon: etherwan
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.2736.1.1
```

Comment:
We have nothing in stats for this OID, I'd be inclined to change it to `.1.3.6.1.4.1.2736.`